src/index.html

```diff
@@ -8,9 +8,54 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="POST"
+      class="form big__form"
+      data-qa="big"
+    >
+      <label class="big__label-search" for="big__search">
+        <img
+          class="big__search-icon"
+          src="images/Search.svg"
+          alt="search bar"
+        >
```

Comment:
Better to use background image for decorative icon

Reply:
fixed